lib/puma/binder.rb

```diff
@@ -48,7 +48,7 @@ def initialize(log_writer, conf = Configuration.new)
 
       @envs = {}
       @ios = []
-      localhost_authority
+      # localhost_authority
```

Comment:
Remove this line? Or was it a mistake to comment this out?

Reply:
Not a mistake.  Can't quite remember the particulars, but `localhost_authority` will fail if some  conditions aren't met.  There's no reason to call it if the 'binds' do not include an ssl connection.  I commented it out because I wanted to run the test suite several times, and forgot to remove the line.

Reply:
It may have happened when running a single file locally, can't recall.